ImprovedAnvil/lib/items/05_replacements.tph

```diff
@@ -73,4 +73,16 @@ COPY_EXISTING ~scrl1e.itm~ ~override/scrl6g.itm~
 
 COPY_EXISTING ~scrlam.itm~ ~override/scrl9j.itm~
 
-COPY_EXISTING ~scrl1x.itm~ ~override/scrl5j.itm~
\ No newline at end of file
+COPY_EXISTING ~scrl1x.itm~ ~override/scrl5j.itm~
+
+COPY_EXISTING_REGEXP GLOB ~.+\.itm~ ~override~
```

Comment:
something weird with the tabs here and in a few other places
I maintain soft tabs sized at 2 spaces

Reply:
True, fixed that